package.json

```diff
@@ -22,7 +22,8 @@
     "next-absolute-url": "^1.1.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "reactstrap": "^8.2.0"
+    "reactstrap": "^8.2.0",
+    "styled-components": "^5.0.0"
```

Comment:
now you can move all style.jsx to styled-components :)

Reply:
True!

Reply:
I've open a specific issue for this one: [issue 11](https://github.com/SocialGouv/carnets/issues/11)